extensions/wikia/Staging/js/Staging.js

```diff
@@ -6,13 +6,15 @@ if ( window.wgStagingEnvironment ) {
 			stagingURLPart = '://' + stagingEnvName + '.',
```

Comment:
@rogatty we had that discussion offline and I think current implementation includes outcome of that talk.